src/services/selfHosted/useSelfHostedSeatsAndLicense.ts

```diff
@@ -19,28 +21,34 @@ export const SelfHostedLicenseSchema = z
 
 export interface UseTierArgs {
   provider: string
+  opts?: QueryOptions
```

Comment:
Can you limit these options to just what you require, i.e. `enabled` in this case. There are typing issues if you just pass the whole options for instance `select` is totally broken when just using the standard `QueryOptions` type because there is no type information for the return type.

Reply:
I do see a fix for this in the future tho ...